qiita_db/portal.py

```diff
@@ -40,10 +40,12 @@ def __init__(self, portal):
     @staticmethod
     def list_portals():
         """Returns list of portals available in system
+
         Returns
         -------
         list of str
             List of portal names for the system
+
         Notes
         -----
         This does not return the QIITA portal in the list, as it is a required
```

Comment:
Doesn't this imply that the short docstring is inaccurate? Is there a benefit to this method not including "qiita"? The argument to include it is that it is a valid portal.

I realize this is outside of the present PR, so can divert to an issue if desired.